src/VisualStudio/Core/Def/SymbolSearch/VisualStudioSymbolSearchService.cs

```diff
@@ -78,7 +78,7 @@ protected override void StartWorking()
         {
             // Always pull down the nuget.org index.  It contains the MS reference assembly index
             // inside of it.
-            Task.Run(() => UpdateSourceInBackgroundAsync(SymbolSearchUpdateEngine.NugetOrgSource));
+            Task.Run(() => UpdateSourceInBackgroundAsync(SymbolSearchUpdateEngine.NugetOrgSourceUri));
```

Comment:
so, this is the bug. this gives Uri which will make _sourceToDatabase to have its key with uri, but the call above _sourceToDatabase.TryGetValue gets NugetOrgSourceName. so it will never be found.